src/css/neo4j-docs.css

```diff
@@ -0,0 +1,185 @@
+:root {
```

Comment:
Could you please add a Fabric, Entreprise Edition and Deprecated pages in the `preview-src`?